plugins/permission/types.ts

```diff
@@ -27,7 +27,7 @@ export type ModularSignerParams = {
     signerContractAddress?: Address
 }
 export type PolicyParams = {
-    policyAddress?: Address
+    policyAddress: Address
```

Comment:
I think let's do versioning of policy just like plugin and the param can be `version`. Since contract don't really have a versioning let's start versioning from `0.0.1`, `0.0.2` and so on...

Reply:
So we keep the `policyAddress` param as it is and option. and add one more required param `policyVersion`
Check how the `validatorAddress` is handled [here](https://github.com/zerodevapp/sdk/blob/main/plugins/ecdsa/toECDSAValidatorPlugin.ts#L68-L72) in ecdsa plugin.